lib/v2/screens/explore_screens/invite/invite_screen.dart

```diff
@@ -0,0 +1,90 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:seeds/v2/blocs/rates/viewmodels/rates_bloc.dart';
+import 'package:seeds/v2/components/alert_input_value.dart';
+import 'package:seeds/v2/components/amount_entry_widget.dart';
+import 'package:seeds/v2/components/balance_row.dart';
+import 'package:seeds/v2/components/flat_button_long.dart';
+import 'package:seeds/v2/components/full_page_error_indicator.dart';
+import 'package:seeds/v2/components/full_page_loading_indicator.dart';
+import 'package:seeds/v2/domain-shared/page_state.dart';
+import 'package:seeds/v2/domain-shared/ui_constants.dart';
+import 'package:seeds/v2/design/app_theme.dart';
+import 'package:seeds/v2/screens/explore_screens/invite/interactor/viewmodels/bloc.dart';
+
+/// INVITE SCREEN
```

Comment:
Do we need comments? I think the class names are self explanatory

Reply:
Not really I just put it because of habit.

It happens that the first time you present the bloc pattern as a new proposal, I put these comments in each part so that it could be understood.